backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidInput.java

```diff
@@ -612,6 +630,8 @@ public boolean isButtonPressed (int button) {
 	final float[] R = new float[9];
 	final float[] orientation = new float[3];
 
+	
+//TODO: ?

```

Comment:
Sorry about that, the pull request may have been a bit hastey.  I'm finding a few things myself.